packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -211,60 +202,32 @@ class DevFSStringContent extends DevFSByteContent {
   }
 }
 
-/// Abstract DevFS operations interface.
-abstract class DevFSOperations {
-  Future<Uri> create(String fsName);
-  Future<dynamic> destroy(String fsName);
-  Future<dynamic> writeFile(String fsName, Uri deviceUri, DevFSContent content);
-}
-
-/// An implementation of [DevFSOperations] that speaks to the
-/// vm service.
-class ServiceProtocolDevFSOperations implements DevFSOperations {
-  ServiceProtocolDevFSOperations(this.vmService);
-
-  final vm_service.VmService vmService;
-
-  @override
-  Future<Uri> create(String fsName) async {
-    final vm_service.Response response = await vmService.createDevFS(fsName);
-    return Uri.parse(response.json['uri'] as String);
-  }
-
-  @override
-  Future<dynamic> destroy(String fsName) async {
-    await vmService.deleteDevFS(fsName);
-  }
-
-  @override
-  Future<dynamic> writeFile(String fsName, Uri deviceUri, DevFSContent content) async {
-    throw UnsupportedError('Use the HTTP devFS api.');
-  }
-}
-
 class DevFSException implements Exception {
   DevFSException(this.message, [this.error, this.stackTrace]);
   final String message;
   final dynamic error;
   final StackTrace stackTrace;
 }
 
-class _DevFSHttpWriter {
-  _DevFSHttpWriter(
+class DevFSHttpWriter {
```

Comment:
Can this be `@visibleForTesting`?

Reply:
I think it could actually be made private again

Reply:
Done